src/docs/security-and-privacy-compliance/devops-security-considerations.md

```diff
@@ -322,133 +298,137 @@ Retention is set as follows:
+Teams may request GitHub apps to be associated with their own or all projects in a GitHub organization.  These requests are reviewed by a the Developer Experience team for validity and scope.  These are also shared with Ministry security staff to ensure they are acceptable for connection.
 
 **GitHub Enterprise**
 
 We currently use of GitHub Enterprise.  Contact the Developer Experience team for license information.
-- [GitHub Enterprise user licenses in the BC government](/github-enterprise-user-licenses-bc-government/)
-- https://github.com/enterprises/bcgov-ent/sso
+* [GitHub Enterprise user licenses in the BC government](/github-enterprise-user-licenses-bc-government/)
+- [Province of British Columbia Enterprise’s single sign-on provider](https://github.com/enterprises/bcgov-ent/sso)
 
-------
-### <a name="protected-c-use"></a>DevOps Team recommendations for Protected C data creation/storage/use
+## DevOps Team recommendations for Protected C data creation, storage and use
 
 This is a list of specific considerations for teams creating/storing/using Protected C data.  **This only applies to data on the Emerald Cluster.**
 
 Product teams should be aware that any creation/storage/use of Protected C data in any system should only be considered after discussion with Ministry Security Officer (MISO) and Ministry Privacy Officer (MPO).  They may require additional controls to those listed below.
  
 Protected C data should be encrypted at rest. 
-- The NetApp storage appliance used is NOT encrypted at its core, so this means that associate provisioned volume claims (PVCs) are not encrypted by default.  However, it is possible to request an encrypted volume claim. 
-  - This is done at pvc creation time using the following https://netapp-trident.readthedocs.io/en/stable-v21.07/kubernetes/operations/tasks/backends/ontap/ontap-san/configuration.html?highlight=encryption#configuration-options-for-provisioning-volumes.  
+- The NetApp storage appliance used is **not** encrypted at its core, so this means that associate provisioned volume claims (PVCs) are not encrypted by default.  However, it is possible to request an encrypted volume claim.
+
+  - This is done at [pvc creation time](https://netapp-trident.readthedocs.io/en/stable-v21.07/kubernetes/operations/tasks/backends/ontap/ontap-san/configuration.html?highlight=encryption#configuration-options-for-provisioning-volumes)
   - If data is to be stored in a database, that database should have encryption enabled and keys managed.  
-  - NetApp storage for Emerald is on its own VIP segment – this means it is NOT accessible from other clusters (this is a good thing).
-- Object Storage used CAN be encrypted as part of initial setup.  
+  - NetApp storage for Emerald is on its own VIP segment – this means it is **not** accessible from other clusters (this is a good thing).
```

Comment:
Every clusters storage is on its own VLAN segment that is not route. Not just Emerald.

Reply:
Ajusted.